src/groovy/org/pih/warehouse/importer/CSVUtils.groovy

```diff
@@ -149,4 +150,16 @@ class CSVUtils {
     static CSVPrinter getCSVPrinter() {
         return new CSVPrinter(new StringBuilder(), CSVFormat.DEFAULT)
     }
+
+    /**
+    * Return a CSV separator character based on provided data and expected number of column
```

Comment:
Can you add a comment here about when you expect the `;` as a separator and why.